.github/workflows/rust.yml

```diff
@@ -39,12 +39,6 @@ jobs:
           command: check
           args: --all-targets --all-features --workspace
 
-      - name: Check no features
-        uses: actions-rs/cargo@v1.0.3
-        with:
-          command: check
-          args: --all-targets --no-default-features --workspace
-
```

Comment:
Is there any reason to remove this?

Reply:
no-default-features deactives `std` feature, therefore activating `no_std` and `error_in_core`, which is a nightly feature only.